packages/pwa-kit-runtime/src/ssr/server/build-remote-server.js

```diff
@@ -181,7 +182,20 @@ export const RemoteServerFactory = {
      */
     // eslint-disable-next-line no-unused-vars
     _setupLogging(app) {
-        // Hook for the dev-server
+        app.use(
+            expressLogging(function(tokens, req, res) {
+                return [
+                    `(${res.locals.requestId})`,
+                    tokens.method(req, res),
+                    tokens.url(req, res),
+                    tokens.status(req, res),
+                    tokens['response-time'](req, res),
+                    'ms',
+                    '-',
+                    tokens.res(req, res, 'content-length') || '-'
```

Comment:
What is the reason for the "or" conditional here. Are we expecting that content length might nowtbe available?

That being said this is close to the output format of the existing logger. But it looks like you might end up with a case where you log line ends with "- -". 

Can we possibly leave it in a better state than before and conditionally add the "-" before the content length based on it's existence?

Reply:
There was no good reason, It was to respect the existing logger. The request makes sense to me. 👍